src/micro_http/src/common/headers.rs

```diff
@@ -472,6 +473,51 @@ mod tests {
             .is_err());
     }
 
+    #[test]
+    fn test_parse_header_in_any_whitespace() {
```

Comment:
nit:
```suggestion
    fn test_parse_header_whitespaces() {
```

Reply:
@ioanachirca Thanks for your review! I fixed it.